packages/client/src/pages/Home/index.js

```diff
@@ -117,7 +118,8 @@ class PureHome extends React.Component {
 
     if (!filterObj) return;
 
-    if (isRooms && alreadyFetchingRooms) return setAlreadyFetchingRooms(false);
+    if (isRooms && alreadyFetchingRooms && selectedFolderStore.title)
```

Comment:
There are many more cases when this bug is reproduced - if you go from My documents to settings and press back in the browser, it does not go and does not highlight. If you go from trash to my documents and press the back button, it does not go either.

Reply:
Fix